internal/pkg/cli/secret_init.go

```diff
@@ -442,9 +442,20 @@ func (o *secretInitOpts) targetEnv(envName string) (*config.Environment, error)
 func BuildSecretInitCmd() *cobra.Command {
 	vars := secretInitVars{}
 	cmd := &cobra.Command{
-		Use:     "secret init",
-		Short:   "Create or update an SSM SecureString parameter.",
-		Example: ``, // TODO
+		Use:   "init",
+		Short: "Create or update Copilot-scoped secrets.",
+		Example: `
+Create a secret by following our guided experience. 
+You will be prompted for the secret's name and the values you want for it in each existing environments.
+/code $ copilot secret init
+Create a secret named db-password. 
+You will be prompted for the values you want for db-password in each existing environments.
+/code $ copilot secret init --name db-password
+Create a secret named db-password with values you want in each existing environments. 
+You will be prompted for the values you want for db-password in each existing environments.
+/code $ copilot secret init --name db-password
+Create secrets from input.yml.
+/code $ copilot secret init --cli-input-yaml input.yml`,
```

Comment:
Can we make the help menu slightly more terse:
```
Create a secret with prompts.
/code $ copilot secret init
Create a secret named db-password in multiple environments
/code $ copilot secret init --name db-password --values test=value1,prod=value2
Create a secret named db-password with values you want in each existing environments. 
Create secrets from input.yml, for the format of the yaml file please see <url>.
/code $ copilot secret init --cli-input-yaml input.yml`,
```
We should provide documentation on the format of the file otherwise they don't have any way of guessing :D